lib/project/web-site-project.ts

```diff
@@ -0,0 +1,112 @@
+///<reference path="../.d.ts"/>
+"use strict";
+
+import path = require("path");
+import util = require("util");
+
+import frameworkProjectBaseLib = require("./framework-project-base");
+import helpers = require("../common/helpers");
+
+export class MobileWebSiteProject extends frameworkProjectBaseLib.FrameworkProjectBase implements Project.IFrameworkProject {
+	constructor(private $config: IConfiguration,
+		$fs: IFileSystem,
+		$logger: ILogger,
+		private $projectConstants: Project.IProjectConstants,
+		private $projectFilesManager: Project.IProjectFilesManager,
+		private $templatesService: ITemplatesService,
+		private $resources: IResourceLoader) {
+		super($logger, $fs);
+	}
+
+	public get name(): string {
+		return this.$projectConstants.TARGET_FRAMEWORK_IDENTIFIERS.MobileWebSite;
+	}
+
+	public get capabilities(): IProjectCapabilities {
+		return {
+			build: false,
+			buildCompanion: true,
+			deploy: false,
+			simulate: true,
+			livesync: false,
+			livesyncCompanion: false,
+			updateKendo: false
+		};
+	}
+
+	public get defaultProjectTemplate(): string {
+		return this.$config.DEFAULT_WEBSITE_PROJECT_TEMPLATE;
+	}
+
+	public get liveSyncUrl(): string {
+		return "";
+	}
+
+	public get requiredAndroidApiLevel(): number {
+		return 10;  // 2.3 Gingerbread
+	}
```

Comment:
Yess, we can throw error. This is the advantage of copy-paste development :smile: